### ISSUE TITLE
The output logit tensor is resized twice.

### ISSUE THREAD

**Issue by ChenDirk** [open-mmlab/mmsegmentation#1529]
Hello,
    I find a small bug when I export onnx file. The output logit tensor is resized twice in the onnx file, before softmax opration. The problem is in mmsegmentation/mmseg/models/segmentors/encoder_decoder.py file, in the 'encode_decode' function, the output is resized at the first time(line 75), and the output is resized at the second time in the 'whole_inference' function(line 210). In my mind, they are repetitive and unnecessary.

**Comment by RunningLeon** [comment]
@ChenDirk Hi, it seems you're right. The exporting to ONNX follows original PyTorch pipeline. @MeowZheng Could you explain why there are two resizes here? Whether they can be merged into one?
